lib/components/term.js

```diff
@@ -281,7 +281,11 @@ export default class Term extends PureComponent {
     return (
       <div
         className={css('fit', this.props.isTermActive && 'active')}
-        style={{padding: this.props.padding}}
+        style={{
+          padding: this.props.padding,
+          fontSize: this.props.fontSize,
+          WebkitFontSmoothing: this.props.fontSmoothing ? this.props.fontSmoothing : ''
```

Comment:
I think it is not needed anymore (because of xterm3 integration since `2.0.0-canary.9`)
Can you try without theses lines?